### PR TITLE
matter.js CHIP test implementations

### PR DESCRIPTION
**Pull request by lauckhart**
Implements CHIP tests for matter.js.  A few of these tests do not pass yet, noted with TODOs where applicable.  Tests run with "npm run ci-test", not during normal testing via "npm run test".

Includes changes to DUT implementations in chip-testing/ to make the test instances compatible with new infrastructure.  This code is shared with original method of CHIP testing so that may break temporarily.

Also has a few minor tweaks and fixes to help tests succeed.

### CODE REVIEW

File: packages/model/src/standard/elements/RelativeHumidityMeasurement.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ export const RelativeHumidityMeasurement = Cluster(
     Attribute(
         {
             name: "MeasuredValue", id: 0x0, type: "uint16", access: "R V", conformance: "M",
-            constraint: "minMeasuredValue to maxMeasuredValue", quality: "X P",
+            constraint: "minMeasuredValue to maxMeasuredValuep", quality: "X P",
```
Comment (Apollon77) [inline packages/model/src/standard/elements/RelativeHumidityMeasurement.ts:26]: Is this right or typo?

File: packages/node/src/behaviors/level-control/LevelControlServer.ts [hunk 2]
```diff
@@ -246,10 +246,14 @@ export class LevelControlServerLogic extends LevelControlLogicBase {
         withOnOff: boolean,
         options: TypeFromPartialBitSchema<typeof LevelControl.Options> = {},
     ) {
+        if (rate === 0) {
```
Comment (Apollon77) [inline packages/node/src/behaviors/level-control/LevelControlServer.ts:249]: This change and the OnOff one is in fact 1.4 changes. Dahlias we move out here and do separate pr when we do 1.4?

**Review by Apollon77 — APPROVED** [review]

File: packages/model/src/standard/elements/RelativeHumidityMeasurement.ts (hunk 1 above)
Comment (lauckhart) [reply packages/model/src/standard/elements/RelativeHumidityMeasurement.ts:26]: typo

File: packages/node/src/behaviors/level-control/LevelControlServer.ts (hunk 2 above)
Comment (lauckhart) [reply packages/node/src/behaviors/level-control/LevelControlServer.ts:249]: discussed in discord, leaving as is
